modules/administration-guide/examples/proc_che-upgrading-che-7-41-to-latest-version-on-openshift.adoc

```diff
@@ -0,0 +1,15 @@
+:_content-type: PROCEDURE
+
+:parent-context-of-{prod-prev-id-short}-{prod-last-version-pre-dwo}-to-{prod-id-short}-{prod-ver}-on-openshift: {context}
+
+[id="{prod-prev-id-short}-{prod-last-version-pre-dwo}-to-{prod-id-short}-{prod-ver}-on-openshift_{context}"]
+= Upgrading Che 7.41 to {prod-ver} on Red Hat OpenShift
+
+Upgrading Che 7.41 to {prod-ver} on Red Hat OpenShift requires first upgrading it to Che 7.49.
+
+.Procedure
+
+. Manually upgrade Che 7.41 to 7.49. See the next section.
```

Comment:
Any one module should not reference 'previous', 'next', or similar.
```suggestion
. Manually upgrade Che 7.41 to 7.49. See the xref:examples$proc_{project-context}-manually-upgrading-che-7-41-to-7-49-on-openshift.adoc[] section.
```

Reply:
I'm under the impression that under Fabrice's guidance we should avoid trying to reference included partials (that's why i didn't do it here).
I've tested the suggestion, and it doesn't seem to work:
![image](https://user-images.githubusercontent.com/78564304/178735306-03bab4df-2de7-4d6e-bedc-4e495255e023.png)

Reply:
Oops, apologies. I didn't implement your first suggestion, which is the reason why this one might be failing. Let me try to fix that.

Reply:
If the link doesn't work, don't use a link. The orig. note still stands: do not reference other modules using positional wording.